components/builder-web/app/shared/project-settings/project-settings.component.html

```diff
@@ -1,12 +1,38 @@
 <div class="project-settings-component">
-  <div class="connect" *ngIf="!project && !connecting">
-    <button mat-raised-button color="primary" class="button" (click)="connect()" [disabled]="!hasPrivateKey">
-      Connect a plan file
-    </button>
-    <span class="no-keys" *ngIf="!hasPrivateKey">
-      <hab-icon symbol="add-circle"></hab-icon>
-      <a [routerLink]="['/origins', origin, 'keys']">Add required origin keys</a>
-    </span>
+  <h3>Default Visibility</h3>
+  <p>Set the default visibility of build artificats (.hart files) for the package. Applies to future build artifacts only. To change the visibility of the existing artifacts, see settings for each artifact in <a [routerLink]="['/pkgs', origin, name]">versions</a>.</p>
```

Comment:
Suggest to remove `in <a [routerLink]="['/pkgs', origin, name]">versions</a>"`because the settings for changing the visibility of existing artifact is yet to exist on the artifact settings page ([issue 1189](https://app.zenhub.com/workspaces/ui-team-5cba23a3b14fdc05970c8f87/issues/habitat-sh/builder/1189)). The existing Versions tab doesn't have such functionalities.